sandpack-react/src/components/CodeEditor/index.tsx

```diff
@@ -98,7 +98,12 @@ export const SandpackCodeEditor = forwardRef<CodeMirrorRef, CodeEditorProps>(
       <SandpackStack className={classNames("editor", [className])} {...props}>
         {shouldShowTabs && <FileTabs closableTabs={closableTabs} />}
 
-        <div className={classNames("code-editor", [editorClassName])}>
+        <div
+          aria-labelledby={`${activeFile}-tab`}
+          className={classNames("code-editor", [editorClassName])}
+          id={`${activeFile}-tab-panel`}
```

Comment:
Considering that a page can contain multiple Sandpack instances with similar content, this ID will not be unique to the page. Is there any reason why you need an ID here? There is an alternative to generate unique identifier inside Sandpack

Reply:
yes, this ID is used inside the `FileTabs` component, to add the `aria-controls` attribute. https://github.com/codesandbox/sandpack/blob/e571870afcb0bd3ac1382fd81b1a7de2cca79db1/sandpack-react/src/components/FileTabs/index.tsx#L205 

is there a way I can get a unique ID and pass it between `FileTabs` and `CodeEditor`?

One way I am thinking is to add a unique id with activeFile , what's your opinion ?

Reply:
I just introduced a new hook (`useSandpackId`) to your branch and I think you could use it to generate ids